### PR TITLE
Update pg-entity dependency

### PR DESCRIPTION
**Pull request by Kleidukos**
This PR follows the latest commit on pg-entity, which allows us to use generic deriving and *DerivingVia* to simplify our database models and reduce the surface of error.

### CODE REVIEW

File: shell.nix [hunk 1]
```diff
@@ -16,6 +16,7 @@ in with pkgs;
       hlint
       haskellPackages.apply-refact
       stylish-haskell
+      git
```
Comment (JonathanLorimer) [inline shell.nix:19]: nice, I use the zsh integration so I am always in an impure shell. I tend to forget these important deps

File: src/DB/ContributorCall.hs [hunk 2]
```diff
@@ -1,16 +1,17 @@
-{-# LANGUAGE OverloadedLists #-}
+
```
Comment (JonathanLorimer) [inline src/DB/ContributorCall.hs]: nit: remove whitespace

**Review by JonathanLorimer — DISMISSED** [review]
Wow, sick PR. Gets rid of so much boiler plate. Just 1 nitpicky comment, but I am happy for you to merge anyways.

File: shell.nix (hunk 1 above)
Comment (Kleidukos) [reply shell.nix:19]: I tried to use a pure shell but there was a problem with libz. But yeah, git was missing :-P

**Review by JonathanLorimer — DISMISSED** [review]

**Review by JonathanLorimer — APPROVED** [review]

**Pull request merged**